### PR TITLE
remove global variable from adapter. add tests.

### PR DESCRIPTION
**Pull request by borisjaskerovich**

### CODE REVIEW

**Comment by ghost** [comment]
Build Passed

File: test/spec/modules/timBidAdapter_spec.js [hunk 1]
```diff
@@ -62,5 +62,91 @@ describe('timAdapterTests', function () {
...
+      'data': '',
+      'options': {'withCredentials': false}
+    };
+    const validServerResponse = {
+      'body': {'id': '48640869bd9db94',
+        'seatbid': [
+          {'bid': [{'id': 'sti0a8b9aca-6080-49f6-bd63-ba54439d2359',
+            'impid': '4746fcaa11197f3',
+            'price': 3,
+            'nurl': 'https://eu.stinger-bidder.tech/api/v1/services/win?seller=prebid&price=${AUCTION_PRICE}&bidcur=USD&bidid=sti0a8b9aca-6080-49f6-bd63-ba54439d2359&strict=true',
```
Comment (shlomisut) [inline test/spec/modules/timBidAdapter_spec.js]: remove all these  - this should not be exposed in a public repo

Comment (borisjaskerovich) [reply test/spec/modules/timBidAdapter_spec.js]: do you mean changing the nurl?
i need this test to pass code coverage.

Comment (shlomisut) [reply test/spec/modules/timBidAdapter_spec.js]: Why does it need to be with actual production values ?

Comment (borisjaskerovich) [reply test/spec/modules/timBidAdapter_spec.js]: i will try to change the values

**Comment by ghost** [comment]
Build Passed

File: test/spec/modules/timBidAdapter_spec.js [hunk 2]
```diff
@@ -97,25 +97,25 @@ describe('timAdapterTests', function () {
...
         'seatbid': [
-          {'bid': [{'id': 'sti0a8b9aca-6080-49f6-bd63-ba54439d2359',
-            'impid': '4746fcaa11197f3',
+          {'bid': [{'id': 'id',
+            'impid': 'impid',
             'price': 3,
-            'nurl': 'https://eu.stinger-bidder.tech/api/v1/services/win?seller=prebid&price=${AUCTION_PRICE}&bidcur=USD&bidid=sti0a8b9aca-6080-49f6-bd63-ba54439d2359&strict=true',
-            'adm': '<script type=\"text/javascript\" id=\"main_script\" src=\"https://growfoodsmart.com/sas/player/directPlayer.php?iid=sti0a8b9aca-6080-49f6-bd63-ba54439d2359&sPlatform=hb&activity=hb&plc=placementCodeTest&width=300&height=250&sCampaignID=50106601&sSeller=testpublisherid&creativeID=700&cb=79f1826c-7f5e-44a2-aa59-314ed8b7e470&sCost=3.000&sBid=3.000&sDomain=mediamart.tv&ab=&udevice=desktop&sDeviceType=desktop&uid=1bf1cc21-26db-4c45-8b72-f3e6b1a3008c&ubrowser=chrome&m=MTI4MzM5ODUyODQ1MQ%3D%3D&uOS=windows_10&ot=SCANNER\"></script>',
+            'nurl': 'https://bidder.url/api/v1/?price=${AUCTION_PRICE}&bidcur=USD&bidid=bidid=true',
+            'adm': '<script type=\"text/javascript\" id=\"main_script\" src=\"https://domain.com/Player.php"></script>',
```
Comment (shlomisut) [inline test/spec/modules/timBidAdapter_spec.js]: remove the `main_script`

**Comment by ghost** [comment]
Build Passed

**Comment by ghost** [comment]
Build Failed - please check http://jenkins.thetimmedia.site:8080/job/prebid_pull_request/89/

**Comment by shlomisut** [comment]
retest this please

**Comment by ghost** [comment]
Build Failed - please check http://jenkins.thetimmedia.site:8080/job/prebid_pull_request/91/

**Comment by wester55** [comment]
retest this please

**Comment by ghost** [comment]
Build Passed

**Comment by ghost** [comment]
Build Passed

**Review by shlomisut — APPROVED** [review]

**Pull request merged**